scripts/ananse

```diff
@@ -231,6 +239,18 @@ if __name__ == "__main__":
         action=NegateAction,
         nargs=0,
     )
+    
+    group.add_argument(
+        "--necessary-output",
+        "--detailed-output",
```

Comment:
you added 2 different names for the same variable?
IMO, better stick with a long one, and a abbreviated one.

ex: `--necessary-output` and `-no`